multi-agent/lib/mas/resources/builtin_service.py

```
@@ -0,0 +1,773 @@
+        self._descriptor_repo.delete(rid)
+        if self._builtin_user_config_repo:
+            self._builtin_user_config_repo.delete_all_for_resource(rid)
+
+    # ---------- Listing / schema ----------
+
+    def find_visible(
+        self,
+        *,
+        identity: Optional[Identity],
+        category: Optional[str],
+        type: Optional[str],
```

Ruff (0.16.1)

[error] Function argument `type` is shadowing a Python builtin

(A002)